internal/xds/bootstrap/bootstrap_test.go

```diff
@@ -1194,3 +1198,78 @@ func (s) TestNode_ToProto(t *testing.T) {
 		})
 	}
 }
+
+// Tests the case where the xDS fallback env var is set to false, and verifies
+// that only the first server from the list of server configurations is used.
+func (s) TestGetConfiguration_FallbackDisabled(t *testing.T) {
+	// TODO(easwars): Default value of "GRPC_EXPERIMENTAL_XDS_FALLBACK"
+	// env var is currently false. When the default is changed to true,
+	// explicitly set it to false here.
```

Comment:
optional: explicitly state what the test expects?

Reply:
This whole test is expected to be run with the env var set to false. This is stated in the name of the test and the top-level comment.

Reply:
i meant

```
			origFallbackEnv := envconfig.XDSFallbackSupport
			envconfig.XDSFallbackSupport = false
			defer func() { envconfig.XDSFallbackSupport = origFallbackEnv }()
```

Reply:
Ah sorry. Should have done that. But now, I will just wait for the default value to change before doing that :)